src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -212,17 +219,28 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
 
   private loadConfig(): void {
     this.planService.getProject(this.scenarioConfigId!).subscribe((config) => {
-      const estimatedCost = this.formGroups[2].get('budgetForm.estimatedCost');
-      const maxCost = this.formGroups[2].get('budgetForm.maxCost');
-      const maxArea = this.formGroups[2].get('physicalConstraintForm.maxArea');
-      const excludeDistance = this.formGroups[2].get(
-        'physicalConstraintForm.excludeDistance'
+      const scenarioName = this.nameFormGroup.get('scenarioName');
+      const estimatedCost = this.constraintsFormGroup.get('budgetForm.estimatedCost');
+      const maxCost = this.constraintsFormGroup.get('budgetForm.maxCost');
+      const maxArea = this.constraintsFormGroup.get('physicalConstraintForm.maxArea');
+      const minDistanceFromRoad = this.constraintsFormGroup.get(
+        'physicalConstraintForm.minDistanceFromRoad'
       );
-      const excludeSlope = this.formGroups[2].get(
-        'physicalConstraintForm.excludeSlope'
+      const maxSlope = this.constraintsFormGroup.get(
+        'physicalConstraintForm.maxSlope'
       );
-      const selectedQuestion = this.formGroups[1].get('selectedQuestion');
+      this.excludedAreasOptions.forEach((area: string) => {
+        if (config.excluded_areas) {
+          if (config.excluded_areas![area]) {
```

Comment:
nit: can we combine this into a single `if`?

Reply:
Ope nice catch — had them separated for a different reason and forgot to fix it when the separation wasn't needed anymore. Fixed!